focus_converter_base/focus_converter/conversion_configs/oci/1_resource_type_S001.yaml

```diff
@@ -0,0 +1,4 @@
+plan_name: convert product/resourceId to ResourceType
+conversion_type: rename_column
+column: product/resourceId
+focus_column: ResourceType
```

Comment:
`product/resourceId` maps to **FOCUS ResourceId** column.
Furthermore, if `product/resourceId` is not null and starts with `ocid1`, ResourceType can be resolved from product/resourceId

OCID template: `ocid1.<RESOURCE TYPE>.<REALM>.[REGION][.FUTURE USE].<UNIQUE ID>`

References:

- [OCI Resource Identifiers](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm)

Reply:
Thank you @ijurica, I will modify this and make another commit.